modules/core/src/v2/coins/eos.ts

```diff
@@ -715,6 +715,7 @@ export class Eos extends BaseCoin {
 
         const refundActionData = txAction.data;
         tx.address = refundActionData.owner;
+        tx.amount = '0';
```

Comment:
pre-existing issue, but if you look above, the error messages are for unstake.
line 709: `new Error(`unstake should only have 1 action: ${tx.actions.length} given`);`
line 713: `throw new Error('Invalid or incomplete unstake action data');`

since we're making a change here, could you update them to `refund`?

Reply:
Good catch, updated the error messages.